RecoLocalMuon/GEMRecHit/test/runTestUpgradeGEMME0Reco_v2_cfg.py

```diff
@@ -0,0 +1,164 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I don't think that we need to proliferate configurations serving the same purpose.
Considering that RecoLocalMuon/GEMRecHit/test/runTestUpgradeGEMME0Reco_cfg.py is not maintainable in its current form, I suggest to add all relevant changes from this file there and remove this _v2